thevoid/connection.cpp

```diff
@@ -542,12 +551,17 @@ void connection<T>::print_access_log()
 		m_access_status,
 		m_access_received,
 		m_access_sent,
-		delta);
+		delta,
+		m_receive_time.count(),
+		m_send_time.count());
```

Comment:
Btw, it is declared that blackhole can take all parameters as %s and using simple c++ magic correctly print every argument. But I am not sure it is implemented in version we used (need to be checked).